database/migrations/2016_10_11_200952_drop_file_id_from_reactions.php

```diff
@@ -25,6 +27,8 @@ public function up()
     public function down()
     {
         Schema::table('reactions', function (Blueprint $table) {
+            $table->unique(['file_id', 'drupal_id', 'northstar_id', 'taxonomy_id'], 'file_drupal_ns_taxonomy_unique');
+
             $table->integer('file_id')->index()->comment('The reportback_file.fid that this kudos applies to.');
```

Comment:
ooo can you do it in this order? this looks like it makes the index using a column that doesn't exist yet?

Reply:
This is just the inverse of the `up` migration above it! The actual index is created [a few migrations earlier](https://github.com/DoSomething/rogue/blob/24d7f508159293a8c7ec0e0a3fa73fe1cf75f40e/database/migrations/2016_07_27_172344_create_kudos_table.php#L23).

Reply:
Oh, although I see what you mean now. 🙃 Yeah I'm surprised that works!

Reply:
Yeah the `up` one seems in the opposite order that I'd expect too? I'm surprised it works too!

Reply:
although @DFurnes if you're dropping a column and and index that uses that column, does the order matter?